src/func_DiffEq_definition_cb.jl

```diff
@@ -93,9 +93,9 @@ function LWFBrook90R_updateAmounts_INTS_INTR_SNOW_CC_SNOWLQ!(integrator)
         p_VXYLEM, p_DISPERSIVITY = integrator.p;
 
     ## B) time dependent parameters
-    @unpack p_DOY, p_MONTHN, p_GLOBRAD, p_TMAX, p_TMIN, p_VAPPRES, p_WIND, p_PREC,
+    @unpack p_DOY, p_MONTHN, p_GLOBRAD, p_TMAX, p_TMIN, p_VAPPRES, p_WIND, p_PREC, p_IRRIG,
```

Comment:
`p_IRRIG`, `p_δ18O_IRRIG`, `p_δ2H_IRRIG` can now be used in the callback function.